### ISSUE TITLE
Figure 2 explanation

### ISSUE THREAD

**Issue by sofiaele** [ZhikangNiu/A-DMA#6]
### Checks

- [x] This template is only for research question, not usage problems, feature requests or bug reports.
- [x] I have thoroughly reviewed the project documentation and read the related paper(s).
- [x] I have searched for existing issues, including closed ones, no similar questions.
- [x] I am using English to submit this issue to facilitate community communication.

### Question details

Hello,

I would like to better understand the exact experimental configuration used for the dual-modality alignment results shown in Figure 2.

Could you please clarify:

- Which diffusion backbone was used for this experiment?
- At which specific layers (or blocks) was the text-guided alignment applied?
- At which specific layers was the speech-guided alignment applied?
- Which SSL speech model was used for the speech-guided alignment?

Any additional implementation details (e.g., whether this corresponds to the default A-DMA configuration or a specific ablation setting) would also be helpful.

Thank you.

**Comment by ZhikangNiu** [comment]
Q1: Which diffusion backbone was used for this experiment?
F5TTS Small, backbone is DiT
Q2: At which specific layers (or blocks) was the text-guided alignment applied?
Text-guided alignment is applied at the 8th layer
Q3: At which specific layers was the speech-guided alignment applied?
12th layer
Q4: Which SSL speech model was used for the speech-guided alignment?
Hubert Large

More details can be found at https://arxiv.org/pdf/2505.19595
@sofiaele

**Comment by sofiaele** [comment]
@ZhikangNiu Thank you for the clarification.

I am currently trying to reproduce the dual-modality alignment results shown in Figure 2, but so far the performance in my experiment appears to be noticeably worse. To verify that I am using the same setup, I am sharing my current configuration and the results after 50k training steps.

````
hydra:
  run:
    dir: ckpts/${model.name}_${model.mel_spec.mel_spec_type}_${model.tokenizer}_${datasets.name}/${now:%Y-%m-%d}/${now:%H-%M-%S}

datasets:
  name: LibriTTS_100_360_500  # dataset name
  dataset_root: /home/data/LibriTTS  # dataset root path
  feature_root: /home/features/hubert_feature # feature root path
  batch_size_per_gpu: 160000 # 2 GPUs, 2 * 160000 = 320000
  batch_size_type: frame  # frame | sample
  max_samples: 64  # max sequences per batch if use frame-wise batch_size. we set 32 for small models, 64 for base models
  num_workers: 16

optim:
  epochs: 100
  learning_rate: 7.5e-5
  num_warmup_updates: 20000  # warmup updates
  grad_accumulation_steps: 1  # note: updates = steps / grad_accumulation_steps
  max_grad_norm: 1.0  # gradient clipping
  bnb_optimizer: False  # use bnb 8bit AdamW optimizer or not

model:
  name: F5TTS_v1_Small  # model name
  tokenizer: char  # tokenizer type
  tokenizer_path: null  # if 'custom' tokenizer, define the path want to use (should be vocab.txt)
  backbone: DiT_ADMA
  arch:
    dim: 768
    depth: 18
    heads: 12
    ff_mult: 2
    text_dim: 512
    text_mask_padding: False
    conv_layers: 4
    pe_attn_head: 1
    checkpoint_activations: True  # recompute activations and save memory for extra compute
  mel_spec:
    target_sample_rate: 24000
    n_mel_channels: 100
    hop_length: 256
    win_length: 1024
    n_fft: 1024
    mel_spec_type: vocos  # vocos | bigvgan
  vocoder:
    is_local: False  # use local offline ckpt or not
    local_path: null  # local vocoder path

ckpts:
  logger: wandb  # wandb | tensorboard | null
  log_samples: True  # infer random sample per save checkpoint. wip, normal to fail with extra long samples
  save_per_updates: 50000  # save checkpoint per updates
  keep_last_n_checkpoints: -1  # -1 to keep all, 0 to not save intermediate, > 0 to keep last N checkpoints
  last_per_updates: 5000  # save last checkpoint per updates
  save_dir: ckpts/${model.name}_${model.mel_spec.mel_spec_type}_${model.tokenizer}_${datasets.name}
````

Results at 50k steps
WER: 23.95%
SIM: 42.14%

These results seem worse than those reported in Figure 2.
Any guidance would be very helpful for reproducing the reported results.

**Comment by ZhikangNiu** [comment]
See https://github.com/ZhikangNiu/A-DMA/blob/main/src/f5_tts/configs/F5TTS_v1_Small_adma.yaml

**Comment by sofiaele** [comment]
@ZhikangNiu I’ve looked at that as well, and my configuration is almost identical. The only differences are that I’m using a slightly larger batch size (320,000 vs. 307,200) and I have checkpoint_activations: True enabled. I wouldn’t expect these factors alone to lead to different results, though.

**Comment by ZhikangNiu** [comment]
<img width="1400" height="922" alt="Image" src="https://github.com/user-attachments/assets/60a9fa7f-9320-4930-b01f-42a5ddf2ae33" />

Does this metric look similar to the one in the figure when updating 50k?

**Comment by ZhikangNiu** [comment]
Table 1 results are updated 400k steps
